ui/pages/send/send-content/send-amount-row/send-amount-row.container.js

```diff
@@ -5,12 +5,21 @@ import {
   sendAmountIsInError,
   getSendAsset,
 } from '../../../../ducks/send';
+import { getMetaMaskAccounts } from '../../../../selectors';
 import SendAmountRow from './send-amount-row.component';
 
 export default connect(mapStateToProps, mapDispatchToProps)(SendAmountRow);
 
 function mapStateToProps(state) {
+  const {
+    metamask: { nativeCurrency },
+  } = state;
+  const { metamask } = state;
```

Comment:
```suggestion
```

Reply:
fixed in 9f54bd84a91fb9d310746a53230f2ebdc9ecfe50

Reply:
- Verified